grin/app/GrinCLI.hs

```diff
@@ -106,19 +135,10 @@ main = do
         opts = defaultOpts { _poOutputDir = outputDir, _poFailOnLint = True }
     case steps of
       [] -> void $ optimize opts program prePipeline postPipeline
-      _  -> void $ pipeline opts program steps
+      _  -> void $ pipeline opts (Just content) program steps
```

Comment:
What is the content for?

Reply:
It is used by the type annotation parser.